File: project2/Team1/spring/src/main/java/bitc/full502/spring/dto/LodgingListDto.java
package bitc.full502.spring.dto;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class LodgingListDto {
    private Long id;
    private String name;
    private String city;
    private String town;
    private String addrRd;   // 도로명 주소(없으면 지번 주소로 대체 가능)
          // 노출용 대표가/최저가 (백엔드 basePrice 값 매핑)
    private String img;
    private Long basePrice;
}

File: project2/Team1/android/app/src/main/java/bitc/fullstack502/android_studio/network/dto/LodgingListDto.kt
package bitc.fullstack502.android_studio.network.dto

data class LodgingListDto(
    val id: Long,
    val name: String?,
    val city: String?,
    val town: String?,
    val addrRd: String?,
    val basePrice: Long?,
    val img: String?
)
